content/participant/jatinkumar.md

```diff
@@ -1 +1,5 @@
-
```

Comment:
@RR-08 is this someone else's file, which you have fixed, or something else?

Reply:
Yes I have accidentally fixed someone else's file. Do I need to revert back for my pr to count?

Reply:
I think so. @RR-08